common/api_adapter.js

```diff
@@ -15,6 +15,7 @@ function get(method, url, authorization) {
         req.set('authorization', authorization);
     }
 
+    console.log('Making Request: ' + JSON.stringify(req));
```

Comment:
Yes, this was intentional.  I was struggling to debug this issue when it occurred because I needed to see the full request that was being made including the Headers.